docs/proposals/19-monitoring.md

```diff
@@ -0,0 +1,373 @@
+---
+title:  Monitoring Stack - Migrating to the prometheus-operator
+gep-number: 0019
+creation-date: 2022-06-21
+status: implementable
+authors:
+- "@wyb1"
+- "@istvanballok"
+reviewers:
+- ""
+---
+
+# GEP-19: Monitoring Stack - Migrating to the prometheus-operator
+
+## Table of Contents
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+- [Alternatives](#alternatives)
+
+## Summary
+
+As Gardener has grown, the monitoring configuration has also evolved with it.
+Many components must be monitored and the configuration for these components
+must also be managed. This poses a challenge because the configuration is
+distributed across the Gardener project among different folders and even
+different repositories (for example extensions). While it is not possible to
+centralize the configuration, it is possible to improve the developer experience
+and improve the general stability of the monitoring. This can be done by
+introducing the [prometheus-operator]. This operator will make it easier for
+monitoring configuration to be discovered and picked up with the use of the
+[Custom Resources][prom-crds] provided by the prometheus-operator. These
+resources can also be directly referenced in Go and be deployed by their
+respective components, instead of creating `.yaml` files in Go, or templating
+charts. With the addition of the prometheus-operator it should also be easier to
+add new features, such as Thanos.
+
+## Motivation
+
+Simplify monitoring changes and extensions with the use of the
+[prometheus-operator]. The current extension contract is described
+[here][extension-contract]. This document aims to define a new contract.
+
+Make it easier to add new monitoring features and make new changes. For example
+using the [prometheus-operator]
+
+The prometheus-operator handles validation of monitoring configuration. It will
+be more difficult to give prometheus invalid config.
+
+### Goals
+
+- The [prometheus-operator] and the [Custom Resource
+  Definitions][prom-crds]
+  will be deployed in each seed cluster.
```

Comment:
IMO this should not be a goal but rather a way (or step) to achieve a goal. A goal should be what we want to achieve with migrating the current solution to the new one (from the above section I assume better supportability, less complexity, a missing feature in the current solution, etc). A goal should rather tackle a problem, missing important functionality, etc.
I think this comment is applicable also for some of the other goals.